docs/rule/kotlin_test.soy

```diff
@@ -0,0 +1,118 @@
+{namespace kotlin_test}
+
+/***/
+{template .soyweb}
+  {call buck.page}
+    {param title: 'kotlin_test()' /}
+    {param navid: 'rule_kotlin_test' /}
+    {param prettify: true /}
+    {param description}
+      A rule that is used to define a set of kotlin files that contain tests.
+    {/param}
+    {param content}
+
+{call buck.rule}
+{param status: 'FROZEN' /}
+{param overview}
+A <code>kotlin_test()</code> rule is used to define a set of
+{sp}<code>.kt</code> files that contain tests to run via JUnit.
+{/param}
+
+{param args}
+
+{call buck.arg}
+  {param name: 'name' /}
+  {param desc}
+  The name of the rule.
+  {/param}
+{/call}
+
+{call buck.arg}
+  {param name: 'srcs' /}
+  {param default : '[]' /}
+  {param desc}
+  Like <a href="kotlin_library.html"><code>kotlin_library()</code></a>,
+  all of the <code>.kt</code> files specified by the
+  {sp}<code>srcs</code> argument will be compiled when this rule is
+  built. In addition, all of the corresponding <code>.class</code>{sp}
+  files that are built by this rule will be passed as arguments to
+  JUnit when this rule is run as a test. <code>.class</code> files
+  that are passed to JUnit that do not have any methods annotated with
+  {sp}<code>@Test</code> are considered failed tests, so make sure that
+  only test case classes are specified as <code>srcs</code>. This is
+  frequently done by specifying <code>srcs</code> as
+  {sp}{literal}<code>glob(['**/*Test.kt'])</code>{/literal}.
+  {/param}
+{/call}
+
+{call buck.arg}
+  {param name: 'resources' /}
+  {param default : '[]' /}
+  {param desc}
+  Same as <a href="kotlin_library.html"><code>kotlin_library()</code></a>.
+  {/param}
+{/call}
```

Comment:
It would be easier for the end user if you had the text here.  You can create a new `__kotlin_common.soy` file and put common arguments in there and just call the templates.